packages/core/graphql/package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "dotenv": "^16.0.3",
     "graphql": "^16.6.0",
+    "graphql-request": "^6.0.0",
```

Comment:
This was a missing dependency?

Reply:
Yes, the generated file uses `graphql-request`, this was throwing a warning on build-time